fetch.bs

```diff
@@ -5356,8 +5359,8 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
          <li><p>If <var>bytes</var> is failure, then <a lt=terminated for=fetch>terminate</a> the
          ongoing fetch.
 
-         <li><p><a for=ReadableStream>Enqueue</a> a {{Uint8Array}} wrapping an {{ArrayBuffer}}
-         containing <var>bytes</var> into <var>stream</var>.
+         <li><p><a for=ReadableStream>Enqueue bytes</a> into <var>stream</var> given
+         <var>bytes</var>.
```

Comment:
There is an existing problem that this needs to queue a task. I can PR that, but this will have to be rebased then.

Reply:
Re-reading this section it feels like most of the section needs to take place in a task. (I.e., everything dealing with _stream_.) Just the reading of the bytes from the network needs to be done in parallel.